docs/chapters/sdk/template-service.rst

```diff
@@ -0,0 +1,19 @@
+Template Service
+================
+
+A template service serves as an example CMake_ project for service developers that should be used as a starting point while creating platform services. To acquire the template service source code the user needs to fetch the code from github.
```

Comment:
I'd be open for the option to have the template repo itself be the structure definition and just referenced as such. Otherwise, if we want to describe it, we do have a description and guideline for a typical repo structure on the internal wik. If that's reproduced/moved to this PR we should ofc deprecate that internal thing.